client/my-sites/post-type-list/index.jsx

```diff
@@ -46,6 +46,7 @@ class PostTypeList extends Component {
 		super( ...arguments );
 
 		this.renderPostRow = this.renderPostRow.bind( this );
+		this.cellRendererWrapper = this.cellRendererWrapper.bind( this );
```

Comment:
If we assume `this` refers to the current instance during a `render` and all these methods are invoked synchronously via `this.`, the `this` reference should remain accurate. It's during callbacks like `onClick={ this.handleClick }` in which we can't necessary trust the `this` context unless explicitly bound. This unclarity is likely why `React.createClass` had automatically bound event handlers, though it was admittedly a bit of hacky magic. I think this is probably fine to keep as-is (being on the safer side, even if not strictly necessary).